waspc/packages/deploy/src/providers/fly/setup/setup.ts

```diff
@@ -61,6 +61,8 @@ async function setupServer(deploymentInfo: DeploymentInfo<SetupOptions>) {
 	// This creates the fly.toml file, but does not attempt to deploy.
 	await $`flyctl launch --no-deploy ${launchArgs}`;
 
+	replaceLineInLocalToml(/min_machines_running = 0/g, 'min_machines_running = 1');
```

Comment:
Might be a bit fragile in case that line `min_machines_running = 0` changes, for example gets some extra white spaces, or that `0` becomes `2` or something else?

In that case, we will just silently fail here, right?

How do we know it will be 0 initially btw, and that this line will even be defined in the file, what if the line is missing?

If solving any of these concerns is too hard and they are not likely, ok, we can leave it as it is, but I would like to understand the situation.

Reply:
This will silently fail if the line changes or it is missing.

I think this is an okay solution for now 🤷‍♂️

We would ideally upgrade our deployment script here to parse the TOML files and editing the parsed object and writing it back to TOML. But this only protects us from content changes, not from Fly API changes -> if they rename the field completely then we are still screwed :D

Reply:
I really don't like silent fails, not when we are trying to make something work a certain way.

So I would suggest that we instead fail loudly here :D. At least we will know it is not behaving as we want it to, and we can then fix it, instead of finding out some time later that things are not happening as we expect them to happen (e.g. that people have servers shutting down when they should be running jobs).

Reply:
> So I would suggest that we instead fail loudly here :D

Imagine the following:
- we implement this so it fails loudly, I guess meaning "stop the deployment" when this line doesn't exist
- everything is good
- imagine then that Fly releases a new version of the TOML file (which they will do at some point)
- all of Wasp users from that point on **can't deploy stuff to Fly via CLI** until we release a new Wasp version

That's why I vote for a silent failure. If the TOML file changes, oh well, users will figure it out at some point. We are just using Fly CLI to deploy stuff, we can't guarantee the setup since we depend on them and their changes.